pom.xml

```diff
@@ -123,6 +131,28 @@
 			<artifactId>javabean-tester</artifactId>
 			<version>1.0.0</version>
 		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/me.ramswaroop.jbot/jbot -->
+		<dependency>
+			<groupId>me.ramswaroop.jbot</groupId>
+			<artifactId>jbot</artifactId>
+			<version>3.0.2</version>
+    		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/net.codebox/javabean-tester -->
+		<dependency>
+			<groupId>net.codebox</groupId>
+			<artifactId>javabean-tester</artifactId>
+			<version>1.0.0</version>
+		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/me.ramswaroop.jbot/jbot -->
```

Comment:
This dependency (`me.ramswaroop.jbot`) is listed twice... we can delete the second one.